.github/workflows/add_comments.yml

```diff
@@ -1,16 +1,17 @@
-name: Greetings
+name: Add comment
 
 on:
 - pull_request_target
 
 jobs:
   greeting:
+    name: Welcome message
     runs-on: ubuntu-latest
     steps:
     - uses: actions/first-interaction@v1
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        pr-message: >
+        pr-message: |
```

Comment:
I *think* this change will improve the formatting so that line breaks are taken into account.  I'd rather have this in its own Markdown file (#1435) but we might need to change to a different action to do that.